controllers/policeStationsController.js

```diff
@@ -0,0 +1,57 @@
+const model = require("../models/model");
+
+exports.getPoliceStations = (req, res) => {
```

Comment:
u can create a repository in which to put all the methods (for convenience n to avoid a repeat )
for example export module.exports = {
method1() {}
method2(){}
}